packages/farrow-schema/src/validator.ts

```diff
@@ -405,6 +411,12 @@ Validator.impl(S.Unknown, {
   validate: (input) => Ok(input),
 })
 
+Validator.impl(S.Never, {
+  validate: (input) => {
+    throw new Error(`${input} is a json, expect 'void' here`)
```

Comment:
`void`is `unit type` in `typescript` , has a value `undefined`. So it may not be a value for `Never type`

```ts
 throw new Error(`${input} is not a valid value for Never which has zero value`)
```